wordle-web/pages/index.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <v-card>
+    <v-card-item>
+      <v-card-title class="text-center" > Welcome to this beautiful website! </v-card-title>
+      <v-card-subtitle>
+        Click the game button... <strong class="text-red-accent-4"> if you dare...</strong>
+      </v-card-subtitle>
+    </v-card-item>
+    <v-card-text>
+      <NuxtLink to="/game"> Game </NuxtLink>
+    </v-card-text>

```

Comment:
Nice use of `NuxtLink` instead of using `router` like we did in class!

Reply:
Thank you! Do you have a preference between the two?